src/lib/components/GroupTree/components/GroupTree.jsx

```diff
@@ -0,0 +1,149 @@
+import * as d3 from "d3";
+import Slider from "../../../shared/slider";
+import GroupTree from "./Plot/group_tree";
+
```

Comment:
Is this file used anywhere? If so, why is it jsx?

Reply:
Well spotted, removed.